osi_trafficcommand.proto

```diff
@@ -490,6 +490,10 @@ message TrafficAction
         // The custom command given to the traffic participant.
         //
         optional string command = 2;
+
+        // The type of the custom command given to the traffic participant.
+        //
+        optional string command_type = 3;
```

Comment:
**Output CCB 09.06.2021:**
1. Please provide more documentation on the "command" and "command_type" messages, stating the difference between them and how/in which situations should it be used.
2. As a  "note", please mention for both commands that they are mapped to OSC.

Reply:
Thanks @kmeids I have updated the comments as suggested.

Does that look OK to you?